packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts

```diff
@@ -0,0 +1,39 @@
+import { SfProject } from '@salesforce/core';
+import { ConfigUtil, workspaceUtils } from '../../../src';
+
+describe('ConfigUtil', () => {
+  describe('getProjectPackageNames', () => {
+    const mockSfProject: any = {
+      getUniquePackageNames: jest
+        .fn()
+        .mockImplementation(() => ['project1', 'project2'])
+    };
+    const sfProjectGetInstanceMock = jest.fn();
+    const FAKE_WORKSPACE = '/here/is/a/fake/path/to/';
+    let getRootWorkspacePathStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      SfProject.getInstance = sfProjectGetInstanceMock;
+      getRootWorkspacePathStub = jest
+        .spyOn(workspaceUtils, 'getRootWorkspacePath')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+
+    it('should return project package directories listed in project config file', () => {
+      // In this test I want to assert that
+      // 1. getRootWorkspace was called first
+      // 2. SfProject.getInstance() was called, passing the value from getRootWorkspace
+      // 3. getUniquePackageNames() was called last
+      // I don't want to test the response from getUniquePackageNames()
```

Comment:
agreed we shouldn't be verifying what is returned from the actual core method. However, I do think it would be valid to verify that the `getProjectPackageNames` method returns the value returned from `SfProject.getInstance(rootWorkspacePath).getUniquePackageNames();` So something like 
```
const fakeProjects = ['project1', 'project2'];
const mockSfProject: any = {
      getUniquePackageNames: jest
        .fn()
        .mockImplementation(() => fakeProjects)
    };
..... after line 30
expect(projectPackageNames).toEqual(fakeProjects);
```